packages/smithy-aws-event-streams/CHANGES.md

```diff
@@ -0,0 +1,4 @@
+## Unreleased
```

Comment:
I assume we should add similar changes.md to other packages and move these to 0.0.1 once released?

Reply:
Yep, was working on that and then started the rename. I'll have 2 more PRs with changes and then finalizing release commits.